File: exercism/python/isogram/isogram.py
#!/usr/bin/python3
# https://exercism.io/my/solutions/55d439aa234544dcb5a8cc75b776dd7b


def is_isogram(string):
    letters = [c for c in string.lower() if c.isalpha()]
    return len(letters) == len(set(letters))

File: exercism/go/isogram/isogram.go
// https://exercism.io/my/solutions/747fe2ed9a704505a22508d80525ef25

package isogram

import (
	"unicode"
)

// IsIsogram determines whether a text is an isogram or not
func IsIsogram(text string) bool {
	appearsInText := make(map[rune]bool)
	for _, c := range text {
		if !unicode.IsLetter(c) {
			continue
		}
		c = unicode.ToLower(c)
		if appearsInText[c] {
			return false
		}
		appearsInText[c] = true
	}
	return true
}
